src/Umbraco.Core/Models/NotificationData.cs

```diff
@@ -0,0 +1,37 @@
+namespace Umbraco.Core.Models
+{
+    public class NotificationData
```

Comment:
@bergmania We should rename this to something like `ContentWithType` , `ContentAndType` (I'm sure @zpqrtbnk might have some ideas too)